client/landing/subscriptions/components/site-subscription-page/site-subscription-details.tsx

```diff
@@ -90,14 +90,38 @@ const SiteSubscriptionDetails = ( {
 		}
 	}, [ paymentDetails ] );
 
-	const subHeaderText = subscriberCount
+	const subscriberCountText = subscriberCount
 		? translate( '%s subscriber', '%s subscribers', {
 				count: subscriberCount,
 				args: [ numberFormat( subscriberCount, 0 ) ],
 				comment: '%s is the number of subscribers. For example: "12,000,000"',
 		  } )
 		: '';
 
+	const hostname = useMemo( () => {
+		try {
+			return new URL( url ).hostname;
+		} catch ( e ) {
+			return '';
+		}
+	}, [ url ] );
+
+	const urlLink = hostname ? (
+		<a href={ url } className="url" rel="noreferrer noopener" target="_blank">
```

Comment:
It might be best practice to use [`ExternalLink`](https://developer.wordpress.org/block-editor/reference-guides/components/external-link/) here.

Reply:
Using ExternalLink in this commit. https://github.com/Automattic/wp-calypso/pull/77954/commits/258e6068a5fa11134f22a48615d1d84ee5ee67fd

![Uploading image.png…]()